src/LibraryManager.Vsix/Commands/InstallLibraryCommand.cs

```diff
@@ -52,21 +52,21 @@ private void BeforeQueryStatus(object sender, EventArgs e)
             }
         }
 
-        private void ExecuteAsync(object sender, EventArgs e)
+        private async void ExecuteAsync(object sender, EventArgs e)
```

Comment:
We should avoid `async void` method which do any substantial work.  Refactor the bulk into an `async Task` method to avoid the pitfalls of `async void`, then await that new method from this one.

This applies to other `async void` methods which also seem to be doing a fair bit of work.